python_modules/dagster/dagster/_core/events/__init__.py

```diff
@@ -1451,11 +1453,16 @@ def __new__(
 
 @whitelist_for_serdes
 class AssetMaterializationPlannedData(
-    NamedTuple("_AssetMaterializationPlannedData", [("asset_key", AssetKey)])
+    NamedTuple(
+        "_AssetMaterializationPlannedData",
+        [("asset_key", AssetKey), ("partition_key", Optional[str])],
```

Comment:
We probably should have used `partition_key` in `AssetMaterialization`, but, now that we've gone with `partition`, I think it's worth being consistent here.